test/e2e/apimachinery/webhook.go

```diff
@@ -789,9 +789,9 @@ func testWebhookForWebhookConfigurations(f *framework.Framework) {
 				Rules: []v1beta1.RuleWithOperations{{
 					Operations: []v1beta1.OperationType{v1beta1.Create},
 					Rule: v1beta1.Rule{
-						APIGroups:   []string{"*"},
-						APIVersions: []string{"*"},
-						Resources:   []string{"*"},
+						APIGroups:   []string{""},
+						APIVersions: []string{"v1"},
+						Resources:   []string{"pods"},
```

Comment:
Maybe change it to some non-exist resources? Intercepting CRUD of "pods" can still break other tests.